lib/Steps/Integrations/NextJs.ts

```diff
@@ -176,25 +181,26 @@ export class NextJs extends BaseIntegration {
     }
   }
 
-  private _fulfillsMinVersion(proposed: string): boolean {
+  private _fulfillsMinVersion(version: string): boolean {
     // The latest version, which at the moment is greater than the minimum
     // version, shouldn't be a blocker in the wizard.
-    if (proposed === 'latest') {
+    if (version === 'latest') {
       return true;
     }
-    const [
-      proposedMajor,
-      proposedMinor = '0',
-      proposedPatch = '0',
-    ] = proposed.split('.');
-    const [minMajor, minMinor, minPatch] = MIN_NEXTJS_VERSION.split('.');
-    if (
-      parseInt(minMajor) > parseInt(proposedMajor) ||
-      parseInt(minMinor) > parseInt(proposedMinor) ||
-      parseInt(minPatch) > parseInt(proposedPatch)
-    ) {
+
+    const cleanedVersion = clean(version);
+    if (cleanedVersion) {
+      // gte(x, y) : true if x >= y
+      return gte(cleanedVersion, MIN_NEXTJS_VERSION);
+    }
+
+    const minVersionRange = `>=${MIN_NEXTJS_VERSION}`;
+    const userVersionRange = validRange(version);
```

Comment:
Does `validRange` return a boolean about whether or not the range is valid, or is it a normalization function (or both)? It's not clear to me whether or not we know by this point that we're dealing with a range.

Reply:
Same answer as before. If it was a boolean, the variable would probably be named `isValidRange` (note the `is` at the beginning). I can add the comments, but I think it has more cons than pros.